core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js

```diff
@@ -77,6 +77,39 @@
     }
   });
 
+  $.fn.dataTable.ext.search.push(
+    function (settings, row, rowIndex) {
+      var match = true;
+      for (var colInd = 0; colInd < row.length; colInd++) {
+        var searchString = $(settings.aoHeader[1][colInd].cell).find('.filter-input').val();
+        if (_.isEmpty(searchString)) { continue; }
+
+        var cellValue = row[colInd];
+        if (settings.aoColumns[colInd].sType === 'num' &&
+          (_.startsWith(searchString, '>') ||
+          _.startsWith(searchString, '<') ||
+          _.startsWith(searchString, '='))) {
+
+          var numValue = parseFloat(cellValue);
+          searchString = searchString.replace(/=/g, '==');
+          searchString = searchString.replace(/&&/g, '&& numValue');
+
+          try {
+            match = eval("numValue" + searchString);
+            if (!match) {
+              break;
+            }
+          } catch (e) {
```

Comment:
this error occurs often enough, e.g. user start typing in filter field ">" or "> 0 && <", all these expressions cannot be evaluated and we'll get an error. idk if it makes sense to log it.